### DISCUSSION TITLE
There should be a "project" to compile

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#562] [Ideas]
This is a very good one I've seen that clearly shows what needs to be done and stabilized before the next version is released, and also serves as a help list so that some talented people can clearly figure out what they can help with.  In this way, problem fixes will be faster and more stable.

https://github.com/orgs/ubports/projects

**Comment by probonopd** [comment]
Right now the showstoppers are the pinned items at the top of https://github.com/helloSystem/ISO/issues

![image](https://github.com/helloSystem/hello/assets/2480569/5a105481-8d77-49d1-9922-fb4acbebb038)
